setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
My vote is for `scitools-iris`, because that's how we always refer to Iris if we need to make it clear it's the SciTools package called Iris, such as we need to here. We've even used this name for Iris [elsewhere in this PR](https://github.com/marqh/iris/blob/9e07587889613e44e331696b4716a484c1c73c64/setup.py#L207)...